tests/cucumber/steps/index.js

```diff
@@ -26,12 +26,19 @@ if (browser) {
   const chrome = require('selenium-webdriver/chrome');
   const firefox = require('selenium-webdriver/firefox');
 
-  const chromeOptions = new chrome.Options();
-  const firefoxOptions = new firefox.Options();
+  let chromeOptions = new chrome.Options();
+  let firefoxOptions = new firefox.Options();
 
   if (process.env.CI) {
-    chromeOptions.addArguments('--no-sandbox', '--headless', '--disable-gpu');
-    firefoxOptions.addArguments('-headless');
+    chromeOptions = chromeOptions.addArguments(
+      'no-sandbox',
+      'disable-gpu',
+      'headless'
+    );
+    firefoxOptions = firefoxOptions
+      .setPreference('remote.active-protocols', 1) // Sets protocol to only WebDriver BiDi
+      .setPreference('network.http.network-changed.timeout', 30) // In Docker tests this apparently matters
```

Comment:
Interesting, I'd expect setting `remote.active-protocols` to 2 (CDP) to be the solution here, since in Firefox 101 they changed the default from 2 to 3 (1|2).

Also, for `network.http.network-changed.timeout`, have you tried more values, or just 30?

Reply:
to be fair 1 was the first one I tried, something about the combination is tripping us.

I did not try more settings for timeout, 30s seemed sufficient, the default is 5